spec/support/map.rb

```diff
@@ -6,7 +6,15 @@ def have_map_with_data(map, attribute, value)
 
 RSpec::Matchers.define :have_token do |token|
   match do |container|
-    container.has_css?(".token[data-token-id='#{token.to_param}']")
+    container.has_css?(".token[data-token-id='#{token.to_param}']", count: 1)
+  end
+end
+
+RSpec::Matchers.define :have_token_with_identifier do |token, identifier|
+  match do |container|
+    container.find(".token[data-token-id='#{token.to_param}']")
+      .find(".token__identifier")
+      .has_content?(identifier)
```

Comment:
Could you specify a failure message for when the token can be found but the identifier cannot?

Reply:
Sure, done in 71db8ef